ephios/plugins/basesignup/templates/basesignup/section_based/fragment_participant.html

```diff
@@ -17,6 +17,13 @@ <h6 class="mb-0">
                     {% endif %}
                 </div>
                 <div>
+                    {% with form.instance.participant|conflicting_shifts:shift as conflicting_shifts %}
+                        {% if conflicting_shifts %}
+                            <span class="fa fa-exclamation-triangle text-danger"
+                                  data-toggle="tooltip"
+                                  title="{{ form.instance.participant }} already participates in {{ conflicting_shifts|join:", " }} at the same time."></span>
+                        {% endif %}
+                    {% endwith %}
```

Comment:
We still need to merge that with the section signup participant fragment...

Reply:
not sure if that is actually possible as they use a different base layout (col-8 vs col-10)

Reply:
also section_based renders the desired section below the participants name which means that just adding another column is not enough

Reply:
ah alright.